code/datums/spells/bloodcrawl.dm

```diff
@@ -131,20 +133,27 @@
 		playsound(get_turf(L), sound, 100, 1)
 		sleep(3 SECONDS)
 
+	var/mindless = (ishuman(victim) || isrobot(victim))
 	if(!victim)
 		to_chat(L, "<span class='danger'>You happily devour... nothing? Your meal vanished at some point!</span>")
 		return
-
-	if(ishuman(victim) || isrobot(victim))
+	if(victim.mind)
 		to_chat(L, "<span class='warning'>You devour [victim]. Your health is fully restored.</span>")
 		L.adjustBruteLoss(-1000)
 		L.adjustFireLoss(-1000)
 		L.adjustOxyLoss(-1000)
 		L.adjustToxLoss(-1000)
+		return
+	if(mindless)
+		to_chat(L, "<span class='warning'>You devour [victim], but their lack of intelligence renders their flesh dull and unappetising, leaving you wanting for more.</span>")
+		L.adjustBruteLoss(-50)
+		if(isslaughterdemon(L))
```

Comment:
is this logic supposed to be reversed?

Reply:
Yes - I missed this during testing. My mistake. Will change it right now.